sdk/core/Azure.Core/tests/ClientDiagnosticsTests.cs

```diff
@@ -305,22 +305,22 @@ public void CreatesASingleListenerPerNamespace()
             Assert.AreEqual(2, testListener.Sources.Count);
         }
 
-        [TestCase(DiagnosticScope.ActivityKind.Internal)]
-        [TestCase(DiagnosticScope.ActivityKind.Server)]
-        [TestCase(DiagnosticScope.ActivityKind.Client)]
-        [TestCase(DiagnosticScope.ActivityKind.Producer)]
-        [TestCase(DiagnosticScope.ActivityKind.Consumer)]
+        [TestCase(ActivityKind.Internal)]
```

Comment:
Wouldn't this fail to build for netcoreapp2.1? Or do we no longer test against that?

Reply:
As far as I know we only test against net6.0, net7.0, and net462, the build target was added to core just so we don't outright break people, I don't think we officially support netcoreapp2.1.

Reply:
`netcoreapp2.1` is long since out of support.   We do not run tests against it, nor do we officially support applications running on it.  Confirming what Maddy mentioned - we haven't removed the explicit Core target to avoid outright breaking things, but we do not offer support for an unsupported framework.